spec/system/errors/errors_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+describe 'GET 500' do
+  it 'displays the page' do
```

Comment:
Nit: This is a really good thing to test. But doing it through the request specs is the preferred place. One for each of the pages to ensure they render the correct template in the `requests/error_spec.rb`:

```ruby
it 'renders the correct template' do
    get '/500'
    expect(response.body).to include("We're sorry, but something went wrong.")
end
```

System tests in Rails are pretty slow and because of that are mostly reserved for testing happy path workflows through the app. Making sure a user can navigate to a course and complete a lesson kind of stuff.